selfdrive/car/chrysler/radar_interface.py

```diff
@@ -51,16 +51,26 @@ def update(self, can_strings):
     vls = self.rcp.update_strings(can_strings)
     self.updated_messages.update(vls)
 
-    if self.trigger_msg not in self.updated_messages:
-      return None
-
     ret = car.RadarData.new_message()
+    ret.parseCompleted = self.trigger_msg in self.updated_messages
+    if ret.parseCompleted:
+      ret.points = self._update_radar_points(self.updated_messages)
+      self.updated_messages.clear()
+    else:
+      ret.points = []
+      ret.errors = self._radar_errors()
```

Comment:
are you never adding any radar errors if parseCompleted?

Reply:
bad indent